tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -33,97 +33,106 @@ public function testFix($expected, $input = null)
         $this->doTest($expected, $input);
     }
 
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     * @param null|string $expectedWithKeepMultipleSpacesAfterComma
+     *
+     * @dataProvider testFixProvider
+     */
+    public function testFixKeepMultipleSpacesAfterComma($expected, $input = null, $expectedWithKeepMultipleSpacesAfterComma = null)
```

Comment:
yes.
then, imagine simple situation, when one will add 2 new configs.
with your current implementation, you will end up with sth like
`$expected, $input`
`$expected, $input, $expectedWithConfigA`
`$expected, $input, $expectedWithConfigA, $expectedWithConfigB`
`$expected, $input, $expectedWithConfigA, $expectedWithConfigB, $expectedWithConfigC`
and then, one could use many options at the same time, so...
`$expected, $input, $expectedWithConfigA, $expectedWithConfigB, $expectedWithConfigC, $expectedWithConfigAB, $expectedWithConfigBC, $expectedWithConfigAC, $expectedWithConfigABC`

then, finally
`$expected, $input, $configuration`